Build/wcDocker.js

```diff
@@ -3626,8 +3626,10 @@ define('wcDocker/frame',[
                 var panel = docker.__create(data.panels[i], this, this.$center);
```

Comment:
What does docker.__create do?

Does it create only the javascript object of the panel?
Does it create any DOM object?

Reply:
I was thinking the same that I should probably check before create.

Reply:
OK. So I debugged the code. The __create will create a docker panel object, which also includes DOM element. I cannot check isLayoutMember before create as it set in create. __restore will then add that panel to docker.